_docs/4-kubernetes/3-bookinfo.md

```diff
@@ -0,0 +1,200 @@
+---
+layout: page
+title: Bookinfo
+permalink: /docs/kubernetes-integration-bookinfo.html
+redirect_from: /docs/kubernetes-integration/bookinfo
+category: Kubernetes Integration
+order: 4
+---
+
+Please refer to [bookinfo](/docs/demo-bookinfo.html) for a detailed description of the
+ application and use case.
+
+> Unlike the original bookinfo sample, this walkthrough does not currently support controlled
+> resiliency testing with Gremlin. These steps may be added in the future.
+>
+> Before attempting to run this sample, please read through the [prerequisites and caveats](/docs/kubernetes-integration-intro.html#prerequisites-caveats)
+
+## Running the Bookinfo Sample Application
+
+1. Bring up the [control plane](/docs/kubernetes-integration-control-plane.html#deploy)
+
+1. Bring up the application containers:
+
+   ```bash
+   $ kubectl create -f examples/k8s-bookinfo.yaml
+   ```
+
+   The above command automatically launches the gateway service and the bookinfo application microservices.
+   ![Bookinfo app](/docs/figures/amalgam8-example-app-bookinfo.svg)
+   Each microservice has a corresponding Kubernetes service associated with it.
+   Except for the *reviews* microservice, all other microservices are backed by a single pod.
```

Comment:
We do have to use replica sets and not replication controllers (outdated). Replica sets are not meant to be used directly (or so I thought). And given that we know k8s way of using deployments is incorrect, do we have to adhere to that definition? In istio, we are using deployments per version to point out the correct way to go about things.